### DISCUSSION TITLE
Colspan and Rowspan attributes

### DISCUSSION THREAD

**Discussion by scott-wojan** [tremorlabs/tremor-npm#89] [Ideas]
https://github.com/tremorlabs/tremor/blob/58a5cc5c843d7f69756f526434fdd8f48e5c4a98/src/components/list-elements/Table/TableRow.tsx#L4

It would be great if you could add rowspan and colspan

**Comment by mitrotasios** [comment]
thanks for raising this, we'll consider it for our next major release!
